docker/dev/compose.yml

```diff
@@ -16,6 +16,8 @@ services:
       - ../../:/app
       - caddy_data:/data
       - caddy_config:/config
+    environment:
+      SERVER_NAME: :80
```

Comment:
Generally, if we're changing the environment + the app's features in the same PR, something went wrong. 🙃